dts/bindings/mdio/xilinx,axi-ethernet-7.-2-mdio.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2024 CISPA Helmholtz Center for Information Security
+# SPDX-License-Identifier: Apache-2.0
+
+# Common fields for MDIO controllers
+
+include: mdio-controller.yaml
+
+compatible: xlnx,axi-ethernet-7.2-mdio
```

Comment:
can you c&p your DT? We don't use this in Linux/U-Boot that's why I would like to know what this is used for.

Reply:
Consider removing the compatible string for the MDIO node to align with the Linux device tree structure.

Additionally, the cva6.dtsi file specifies the axi-ethernet-subsystem node as the parent for both the Ethernet and MDIO nodes. This is not consistent with the Linux device tree hierarchy, where the AXI Ethernet node is the parent, the MDIO node is a child of axiethernet, and the PHY node is a child of the MDIO node.